packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -24,15 +24,18 @@ import React, { type PropsWithChildren } from 'react';
 import { Button, type ButtonProps, DensityProvider, Select } from '@dxos/react-ui';
 import { getSize } from '@dxos/react-ui-theme';
 
+import { type Formatting } from '../../extensions/markdown';
+
 // TODO(burdon): Revert to string to make extensible?
 export type ActionType =
   | 'blockquote'
-  | 'bold'
+  | 'strong'
```

Comment:
I went with the strong/emphasis terminology from the CommonMark spec instead of bold/italic.